client/src/Components/RegisterPageComponents/Stepper/Steps/LoginInformation.jsx

```diff
@@ -0,0 +1,169 @@
+
+import {
+  TextField,
+  InputAdornment,
+  Button,
+  Avatar,
+  FormHelperText,
+} from "@mui/material";
+import { styled } from "@mui/material/styles";
+
+import PersonIcon from "@mui/icons-material/Person";
+import EmailIcon from "@mui/icons-material/Email";
+import { Visibility, VisibilityOff } from "@mui/icons-material";
+
+import PhotoCamera from "@mui/icons-material/PhotoCamera";
+
+import "./style.css";
+
+const Input = styled("input")({
+  display: "none",
+});
+
+const convertToBase64 = (file) => {
+  return new Promise((resolve, reject) => {
+    const fileReader = new FileReader();
+    fileReader.readAsDataURL(file);
+    fileReader.onload = () => {
+      resolve(fileReader.result);
+    };
+    fileReader.onerror = (error) => {
+      reject(error);
+    };
+  });
+};
+
+export default function LoginInformation({ loginInformationfForm }) {
+  const [showPassword, setShowPassword] = useState(false);
+
+  const handleClickShowPassword = () => {
+    setShowPassword(!showPassword);
+  };
+
+  return (
+    <form
+      className="form__container"
+      onSubmit={loginInformationfForm.handleSubmit}
+    >
+      <TextField
+        sx={{ mt: 3, width: "500px" }}
+        id="outlined-basic"
+        label="اسم النادي"
+        name="name"
+        InputProps={{
+          endAdornment: (
+            <InputAdornment position="end">
+              <PersonIcon />
+            </InputAdornment>
+          ),
+        }}
+        value={loginInformationfForm.values.name}
+        onChange={loginInformationfForm.handleChange}
+        error={!!loginInformationfForm.errors.name}
+        helperText={loginInformationfForm.errors.name}
+        variant="outlined"
+      />
+
+      <TextField
+        sx={{ mt: 3, width: "500px" }}
+        label="أدخل البريد الإلكتروني"
+        name="email"
+        InputProps={{
+          endAdornment: (
+            <InputAdornment position="end">
+              <EmailIcon />
+            </InputAdornment>
+          ),
+        }}
+        value={loginInformationfForm.values.email}
+        onChange={loginInformationfForm.handleChange}
+        error={!!loginInformationfForm.errors.email}
+        helperText={loginInformationfForm.errors.email}
+        variant="outlined"
+      />
+
+      <TextField
+        sx={{ mt: 3, width: "500px" }}
+        label="أدخل كلمة السر"
+        type={showPassword ? "text" : "password"}
+        name="password"
+        InputProps={{
+          endAdornment: (
+            <InputAdornment position="end" onClick={handleClickShowPassword}>
+              {showPassword ? (
+                <VisibilityOff sx={{ cursor: "pointer" }} />
+              ) : (
+                <Visibility sx={{ cursor: "pointer" }} />
+              )}
+            </InputAdornment>
+          ),
+        }}
+        value={loginInformationfForm.values.password}
+        onChange={loginInformationfForm.handleChange}
+        error={!!loginInformationfForm.errors.password}
```

Comment:
add autoComplete attribute <a chrome advice>
```suggestion
        error={!!loginInformationfForm.errors.password}
         autoComplete="new-password"

```